vuepress/docs/v6.3.2/docs/create/blueprint-features.md

```diff
@@ -0,0 +1,130 @@
+# Entando Blueprint Features
```

Comment:
I'm confused. Was this added to 6.3.2 just as part of this PR? Was trying to make sense of the diff.